<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsDialog</class>
 <widget class="QDialog" name="PrefsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="guiplayer.qrc">
    <normaloff>:/dmidiplayer.png</normaloff>:/dmidiplayer.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabGen">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,0">
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="chkAutoSongSettings">
         <property name="text">
          <string>Automatically Load and Save Song Settings</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="chkSnapping">
         <property name="text">
          <string>Sticky Window Borders</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="cboStyle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="chkAutoAdvance">
         <property name="text">
          <string>Advance Automatically to the Next Playlist item</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lblReset">
         <property name="text">
          <string>MIDI System Exclusive Reset:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="chkDarkMode">
         <property name="text">
          <string>Forced Dark Mode</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="cboReset"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Solo Button % Volume Reduction:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblPercChannel">
         <property name="text">
          <string>Percussion MIDI Channel:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>spinPercChannel</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinPercChannel">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lblQtStyle">
         <property name="text">
          <string>Qt Widgets Style:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>cboStyle</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="lblHidden">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QCheckBox" name="chkInternalIcons">
         <property name="text">
          <string>Use Internal Icon Theme</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinSoloVol">
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="chkAutoPlay">
         <property name="text">
          <string>Start Playback Automatically After Loading</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="chkKeepSnapped">
         <property name="text">
          <string>Keep Snapped Child Windows Together</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLyr">
      <attribute name="title">
       <string>Lyrics</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1,0">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
       </property>
       <item row="0" column="3">
        <widget class="QToolButton" name="btnTextFont">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblPastColor">
         <property name="text">
          <string>Past Text Color</string>
         </property>
         <property name="buddy">
          <cstring>editPastColor</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lblPast">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="editTextFont">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="editFutureColor">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lblFuture"/>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="btnPastColor">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblFutureColor">
         <property name="text">
          <string>Future Text Color</string>
         </property>
         <property name="buddy">
          <cstring>editFutureColor</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblTextFont">
         <property name="text">
          <string>Text Font</string>
         </property>
         <property name="buddy">
          <cstring>editTextFont</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="btnFutureColor">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="editPastColor">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblHighlightColor">
         <property name="text">
          <string>Highlight Color</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="editHighlightColor">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QToolButton" name="btnHighlightColor">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lblHigh"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPiano">
      <attribute name="title">
       <string>Player Piano</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,1,0">
       <item row="6" column="2" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>138</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="editSingle">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblNoteNames">
         <property name="text">
          <string>Show Note Names</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lblSingle"/>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="btnSingleColor">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblHighlight">
         <property name="text">
          <string>Note Highlighting</string>
         </property>
         <property name="buddy">
          <cstring>cboHighlight</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="editNoteFont">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblSingleColor">
         <property name="text">
          <string>Single Highlight Color</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QToolButton" name="btnNoteFont">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="cboHighlight"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblNoteFont">
         <property name="text">
          <string>Note Names Font</string>
         </property>
         <property name="buddy">
          <cstring>editNoteFont</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QComboBox" name="cboNoteNames">
         <item>
          <property name="text">
           <string>Never</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Minimal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>When Activated</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Always</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QCheckBox" name="chkVelocityColor">
         <property name="text">
          <string>Note Velocity to Color Tint</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QCheckBox" name="chkOctaveSubscript">
         <property name="text">
          <string>Octave Subscript Designation</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok|QDialogButtonBox::StandardButton::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinPercChannel</tabstop>
  <tabstop>spinSoloVol</tabstop>
  <tabstop>chkAutoPlay</tabstop>
  <tabstop>chkAutoAdvance</tabstop>
  <tabstop>chkAutoSongSettings</tabstop>
  <tabstop>chkSnapping</tabstop>
  <tabstop>chkDarkMode</tabstop>
  <tabstop>chkInternalIcons</tabstop>
  <tabstop>cboStyle</tabstop>
  <tabstop>cboReset</tabstop>
  <tabstop>editTextFont</tabstop>
  <tabstop>btnTextFont</tabstop>
  <tabstop>btnFutureColor</tabstop>
  <tabstop>btnPastColor</tabstop>
  <tabstop>cboHighlight</tabstop>
  <tabstop>btnSingleColor</tabstop>
  <tabstop>chkVelocityColor</tabstop>
  <tabstop>editNoteFont</tabstop>
  <tabstop>btnNoteFont</tabstop>
  <tabstop>cboNoteNames</tabstop>
 </tabstops>
 <resources>
  <include location="guiplayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrefsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrefsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
